project-base/app/src/DataFixtures/Demo/ParameterDataFixture.php

```diff
@@ -88,7 +88,6 @@ class ParameterDataFixture extends AbstractReferenceFixture implements Dependent
     public const string PARAM_WARRANTY_IN_YEARS = 'warranty_in_years';
     public const string PARAM_WATER_RESERVOIR_CAPACITY = 'water_reservoir_capacity';
     public const string PARAM_WEIGHT = 'weight';
-    public const string PARAM_WEIGHT_KG = 'weight_kg';
```

Comment:
nitpick regarding the commit message, "unified" is IMHO more appropriate word, see https://thecontentauthority.com/blog/unified-vs-united